README.md

```diff
@@ -84,7 +84,7 @@ pip install -r scripts/requirements.txt
 ### Build
 You can build the documentation in two ways:
 
-* Use the provided script `doc/sphinx_build_local.sh` and open the `doc/build/html/index.html` file to see the output.
+* Use the provided script `doc/generate_sphinx_doc.sh` and open the `doc/build/html/index.html` file to see the output.
```

Comment:
Have you tested that it works same as the previous script?
`venv` not being created doesn't cause any issues?

Reply:
Tested and everything works fine